docs/README.FreeBSD.md

```diff
@@ -72,7 +72,7 @@ git clone https://github.com/xbmc/xbmc kodi
 ## 3. Install the required packages
 If you get a `package not found` type of message with the below command, remove the offending package(s) from the install list and reissue the command. Take a note of the missing dependencies and, after a successful step completion, **[build the missing dependencies manually](#31-build-missing-dependencies)**.
 
-**NOTE:** Kodi requires a compiler with C++14 support, i.e. gcc >= 4.9 or clang >= 3.4
+**NOTE:** Kodi requires a compiler with C++17 support.
```

Comment:
we should keep the note about minimum compiler versions: gcc >= 7, clang >= 5

Reply:
@garbear is there a reason you wanted to remove this?

Reply:
commit was authored Feb 24 2020 while integrating libp2p into Kodi. No idea :slightly_smiling_face: